src/main/java/org/apache/commons/collections4/bloomfilter/BitSetBloomFilter.java

```diff
@@ -25,7 +25,7 @@
 import org.apache.commons.collections4.bloomfilter.hasher.StaticHasher;
 
 /**
- * A bloom filter using a Java BitSet to track enabled bits. This is a standard
+ * A bloom filter uses a Java BitSet to track enabled bits. This is a standard
```

Comment:
Perhaps the grammar should be:
```
A bloom filter **that** uses a Java BitSet
```
Anyway, this is a change totally unrelated to the SimpleBloomFilter in this PR. So should be reverted.

Since the backing implementation is largely irrelevant (aside from allowing the determination of memory requirements) a better wording would then be `A Bloom filter using a fixed size array ...` and the entire class renamed to ArrayBloomFilter. So this change should be done separately.